client/js/components/drawer.js

```diff
@@ -0,0 +1,10 @@
+import showHide from './show-hide';
+
+export default (el) => {
+  const toggle = showHide({
+    el,
+    activeClass: 'is-drawer-open'
+  });
+
+  toggle.trigger.addEventListener('click', toggle.toggleActive);
```

Comment:
I think we dabbled with that idea near the beginning of the project and opted to mixin the showHide component as and when we needed it, but I'd be happy to revisit the idea of a more generic show-hider.